server.js

```diff
@@ -1,8 +1,24 @@
 const express = require("express");
 const app = express();
+const mongoose = require('mongoose');
 
-app.get("/ping", (req, res) => {
-  res.send("Hello world!");
+const uri = "mongodb+srv://Manya_Jain:Gunnu324@cluster1.fvkwdo1.mongodb.net/?retryWrites=true&w=majority";
```

Comment:
Move the database connection to the start of the server and use environment variables for sensitive information like the database URI. [important]

Reply:
hey @Manya324, this is a valid suggestion, kindly consider implementing this.

Reply:
Sure, I will definitely try implementing this!